### PR TITLE
feat: Well component

### PR DESCRIPTION
**Pull request by y-lohse**
Adds a [well component](https://yannick-lohse.fr/cozy-ui/react/#well).

This PR replaces #658 — as @ptbrowne pointed out to me, the `ContextHeader` can be replaced by a combination of existing components, except for the Well that is currently missing.

Once this PR is merged, I'll make another one to remove the `ContextHeader`.

### CODE REVIEW

File: react/Well/styles.styl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@require 'settings/palette.styl'
+@require 'settings/breakpoints'
+@require 'tools/mixins'
+
+.well
+  margin rem(16 32)
+  background paleGrey
+  border rem(1) solid silver
+  padding rem(16)
```
Comment (y-lohse) [inline react/Well/styles.styl]: Should we / can we use utility classes in cozy-ui itself?

**Comment by y-lohse** [comment]
[(8) Déplacer un fichier ou un dossier](https://trello.com/c/OSfXpP1q/847-8-d%C3%A9placer-un-fichier-ou-un-dossier)

**Comment by y-lohse** [comment]
[(2) Déplacer plusieurs fichiers/dossiers](https://trello.com/c/zKuIBwvp/1322-2-d%C3%A9placer-plusieurs-fichiers-dossiers)

File: react/Well/styles.styl (hunk 1 above)
Comment (ptbrowne) [reply react/Well/styles.styl]: Why not ?

**Pull request merged**